api/repositories/service_binding_repository_test.go

```diff
@@ -54,7 +58,9 @@ var _ = Describe("ServiceBindingRepo", func() {
 			userClientFactory.WithWrappingFunc(func(client client.WithWatch) client.WithWatch {
 				return authorization.NewSpaceFilteringClient(client, k8sClient, nsPerms)
 			}),
-			conditionAwaiter)
+			conditionAwaiter,
+			nil,
```

Comment:
Instead of using `nil` here, I would suggest wire the parameters client, basically what you do [here](https://github.com/cloudfoundry/korifi/pull/3816/files#diff-cdff009460ea405b57b56faac218be31449c0dc56f8d004a1a25030688811a12R1186-R1201). Thus you would not need to reinitialise the repository in the nested `Describe`